mongo/options/createcollectionoptions.go

```diff
@@ -131,6 +172,12 @@ func (c *CreateCollectionOptions) SetValidator(validator interface{}) *CreateCol
 	return c
 }
 
+// SetTimeSeriesOptions sets the options for time-series collections.
+func (c *CreateCollectionOptions) SetTimeSeriesOptions(timeSeriesOpts TimeSeriesOptions) *CreateCollectionOptions {
```

Comment:
This can take in a pointer instead since options.TimeSeries() returns a pointer anyway

Reply:
Done.